ramls/accounts.raml

```diff
@@ -66,6 +66,38 @@ resourceTypes:
             body:
               application/json:
                 type: accountsCheckResponse
+          404:
+            description: "Account was not found"
+            body:
+              text/plain:
+                example: "Account was not found"
+          422:
+            description: "Returns accounts check response with validation error"
+            body:
+              application/json:
+                type: accountsCheckResponse
+          500:
+            description: "Internal server error"
+            body:
+              text/plain:
+                example: "Internal server error"
+    /check-waive:
+      post:
+        description: Checks if waive is valid
+        body:
+          application/json:
+            type: accountsCheckRequest
+        responses:
+          200:
+            description: "Returns accounts check response if validation is successful"
```

Comment:
The style of this description is different from others. And, again, we're not checking the account itself.

Reply:
fixed